src/server/utilities/cspHeader/domainLists.js

```diff
@@ -7,6 +7,7 @@ export const bbcDomains = [
   '*.bbc.com',
   '*.bbci.co.uk',
   '*.bbci.com',
+  'http://localhost.bbc.com',
```

Comment:
cc @amoore108 and @eagerterrier: this is the reason why MAP pages were failing on local - it was unable to download media (and other) scripts because of it being on localhost.bbc.com but elsewhere in the CPS it was only permitting self: https

I _think_ this should resolve it once and for all? Are you happy with this change?

Reply:
Do I also need to include the 2 known ports 7080 & 7081 or is this enough?

Reply:
Curious its not captured by this anyway:
https://github.com/bbc/simorgh/blob/3b0714959b5a220064704ac6124609dedc0150dc/src/server/utilities/cspHeader/domainLists.js#L7

Reply:
I think it was this causing the issue: https://github.com/bbc/simorgh/blob/3b0714959b5a220064704ac6124609dedc0150dc/src/server/utilities/cspHeader/directives.js#L294-L296